### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by siddhesh2309**
adding HW1 solutions

### CODE REVIEW

File: homework1/S5.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+
+public class S5 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String s = sc.nextLine();
+
+        boolean flag = Palindrome(s);
```
Comment (parthbhope-kd) [inline homework1/S5.java]: use more meaningful variable names like isPalindrome from next time

File: homework1/S3.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import java.util.Scanner;
+public class S3 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        findEvenOdd();
+    }
+    public static void findEvenOdd() {
+        int N = 1000;
+        for(int i = 1; i < N; i++) {
+            if(i % 3 == 0)
```
Comment (parthbhope-kd) [inline homework1/S3.java]: You are supposed to check if sum of digits is divisible by 3.

**Review by parthbhope-kd — CHANGES_REQUESTED** [review]

File: homework1/S5.java (hunk 1 above)
Comment (siddhesh2309) [reply homework1/S5.java]: Made the necessary changes regarding naming conventions

File: homework1/S3.java (hunk 2 above)
Comment (siddhesh2309) [reply homework1/S3.java]: Changed the code, which was expected

**Pull request merged**